compiler_opt/rl/compilation_runner.py

```diff
@@ -80,14 +80,23 @@ def __exit__(self, exc, value, tb):
     pass
 
 
-def get_workdir_context():
+def get_workdir_context(keep_temps: Optional[str] = None):
   """Return a context which manages how the temperory directories are handled.
 
   When the flag keep_temps is specified temporary directories are stored in
   keep_temps.
+
+  Args:
+    keep_temps: Put temporary files into given directory and keep them
+      past exit when compilining
   """
+  if keep_temps and _KEEP_TEMPS.value:
```

Comment:
I see... _KEEP_TEMPS... maybe we can refactor this thing's name after. But let's name the API clearly.

Or you can rename both in one swoop after, up to you.